ruby/sorting-words-and-numbers/main.rb

```diff
@@ -1,16 +1,19 @@
 # frozen_string_literal: true
+SEPARATER = ' '
 
-loop do
-  sorted = [[], []]
-  line = gets
-  next if line.nil?
+while line = gets&.chomp
```

Comment:
Added `safe navigation operator` because it causes an error when `nil.chomp` is executed when EOF is reached.